### PR TITLE
Add 'mobile' or 'desktop' to conditional filter ID

### PR DESCRIPTION
**Pull request by fhlavac**
Added '-mobile' or '-desktop' to conditional filter ID for QA

@john-dupuy

### CODE REVIEW

File: packages/components/src/ConditionalFilter/ConditionalFilter.js [hunk 1]
```diff
@@ -64,11 +64,11 @@ class ConditionalFilter extends Component {
                                         {
                                             ...activeItem.type !== conditionalFilterType.custom &&
                         {
-                            placeholder: placeholder || activeItem.placeholder || `Filter by ${activeItem.label}`,
-                            id: (activeItem.filterValues && activeItem.filterValues.id) || currentValue
+                            placeholder: placeholder || activeItem.placeholder || `Filter by ${activeItem.label}`
                         }
                                         }
                                         { ...activeItem.filterValues }
+                                        id={`${(activeItem.filterValues && activeItem.filterValues.id) || currentValue}-mobile`}
```
Comment (john-dupuy) [inline packages/components/src/ConditionalFilter/ConditionalFilter.js:71]: Is it possible to add an `ouiaId` to these too?

**Review by rvsia — COMMENTED** [review]
I am not sure if we want to change ID when the component is mobile or not. I guess the ID should always stay the same. Isn't ouiaId enough?

**Comment by fhlavac** [comment]
@john-dupuy is the ouiaId enough?

**Comment by rvsia** [comment]
@fhlavac Oh, there are two inputs at that same time? 👀  Then it probably makes sense to have different IDs. 

It would be nice to show only one of the inputs in DOM, I have a hook for getting PF screen variant, so maybe we could integrate it here: https://github.com/RedHatInsights/sources-ui/blob/master/src/hooks/useScreen.js

**Comment by john-dupuy** [comment]
> It would be nice to show only one of the inputs in DOM

I definitely agree here :+1:

**Comment by Hyperkid123** [comment]
@fhlavac OK We can make it show only one. I wanted to avoid having additional listeners but I guess it is optimal to have just one. @fhlavac I will close this PR and create a new one with only one input in the DOM.

**Pull request closed without merging**